### DISCUSSION TITLE
Images in Labels and Titles

### DISCUSSION THREAD

**Discussion by Nanooonym** [ditdot-dev/vue-flow-form#157] [Ideas] [answered]
Hi ! Don'tknow if this project is still supported, but will ask the question anyway :
Is there any way to display images as question title, or as label in questions ? (thinking about multiple choice questions where you choice one or many images). If not, I think that would be a great implementation !

**Comment by spinn** [comment]
Hi! The project is indeed still supported, both for bugfixes and feature requests. Thanks for your suggestion, we think it's a great idea and will be coming in the next update.

**Comment by spinn** [comment, marked as answer]
Support for pictures in multiple choice questions has been added in c65e5a25a99dd0e77c5eca809ad73111ae8f38bb. Example usage:

```js
  ...
  new QuestionModel({
    type: QuestionType.MultiplePictureChoice,
    options: [
      new ChoiceOption({
        imageSrc: '...',
        imageAlt: '...'
      }),
      new ChoiceOption({
        title: '...'
      })
    ]
  })
  ...
```

We haven't yet pushed the new version to npm but we'll do this shortly as well.
